scripts/main.js

```diff
@@ -71,4 +71,94 @@ const departments = {
     }
 }
 
-console.log(departments);
\ No newline at end of file
+console.log(departments);
+
+// Opdracht 1 - Gegevens aanspreken
+
+console.log('De afdeling Sales heeft ' + departments.sales.numberOfEmployees + ' medewerkers');
+console.log('Marketing is een leuke afdeling om te werken. ' +departments.marketing.description);
+console.log('De afdeling Customer Service heeft ' +departments["customer-service"].numberOfEmployees + ' medewerkers');
+console.log('Sales is een uitdagende afdeling om te werken als Verkoopmanager.  ' +departments.sales.description);
+
+// Opdracht 2 - prompten en beslissen
+
+let gekozenAfdeling = prompt('Over welke afdeling wil je meer informatie? Kies uit: [marketing / sales / customer-service]');
+
+if (gekozenAfdeling === 'marketing') {
```

Comment:
Deze had je zelf al opgemerkt, dat lijkt in de uitwerkingen wat overzichtelijker. Daarin is het gefixt dmv _switch_ _case_ en _break_-statements. 

- Switch statements kunnen worden toegepast in situaties waar specifieke, vooraf bepaalde opties zijn. 
- Binnen een switch statement, geven case statements de verschillende opties aan die de switch kan behandelen. 
- Het break statement wordt gebruikt om het einde van een case te markeren en te voorkomen dat de uitvoering doorgaat naar de volgende case.